index.js

```diff
@@ -26,7 +26,7 @@ module.exports = class PassportTrailpack extends Trailpack {
       return Promise.reject(new Error('No strategies found at config.passport.strategies !'))
     }
 
-    if (strategies.jwt && _.get(this.app, 'config.passport.jwt.tokenOptions.secret') === 'mysupersecuretoken') {
+    if (strategies.jwt && _.get(this.app, 'config.passport.jwt.tokenOptions.secret') === 'mysupersecuretoken') {  //eslint-disable-line
```

Comment:
My bad, it's a code habit
They're max-len warnings, generally, after taking note of these, I disable the warning for keeping jslint report clean, I remove the disable-line